### PR TITLE
snap: added snap build files

### PR DESCRIPTION
**Pull request by soumyaDghosh**
Hii, I am extremely sorry, I am so late to push the snap build files. This is the PR. Thanks

### CODE REVIEW

File: snap/snapcraft.yaml [hunk 1]
```diff
@@ -1,36 +1,87 @@
...
-      - dbus-saber
-
+name: saber # you probably want to 'snapcraft register <name>'
+base: core22 # the base snap is the execution environment for this snap
+adopt-info: saber
+grade: stable # must be 'stable' to release into candidate/stable channels
+confinement: strict # use 'strict' once you have the right plugs and slots
+license: GPL-3.0
+compression: lzo
+icon: icon.png
```
Comment (adil192) [inline snap/snapcraft.yaml:8]: I see that you've deleted `snap/gui/saber.png`. Where does this `icon.png` come from?

Comment (soumyaDghosh) [reply snap/snapcraft.yaml]: Hi thanks for pointing it out. I forgot to update the path. This is the new updated path of icon.

Comment (adil192) [reply snap/snapcraft.yaml]: Weren't you using a workflow for the snap builds in your old repo, or was that just handled by snap?

Comment (soumyaDghosh) [reply snap/snapcraft.yaml]: No, it's there just to have it tested. Nothing extra. It's done by connecting this github repo directly to the snapcraft builder in the `Builds` option of the [`Snapcraft Builder`](https://snapcraft.io/saber/builds)

Comment (adil192) [reply snap/snapcraft.yaml]: Ah okay, yeah a workflow would be a nice assurance :)

**Comment by soumyaDghosh** [comment]
Hii, I have tried to add workflow, but I am not good in using git, so, kindly check if everything is fine.

**Comment by adil192** [comment]
In https://github.com/soumyaDghosh/saber you have:
```yaml
     - uses: diddlesnaps/snapcraft-review-action@v1
        with:
          snap: ${{ steps.build.outputs.snap }}
          isClassic: 'false'
```

Should we add this to this repo's workflow

**Comment by soumyaDghosh** [comment]
> In https://github.com/soumyaDghosh/saber you have:
> 
> ```yaml
>       - uses: diddlesnaps/snapcraft-review-action@v1
>         with:
>           snap: ${{ steps.build.outputs.snap }}
>           isClassic: 'false'
> ```
> 
> Should we add this to this repo's workflow

No need, cause it'll always fail. It doesn't have any api yet, to check if the dbus is granted to it or not. The v2 is under development.

**Comment by soumyaDghosh** [comment]
Hii... merge this please and update the snap also

**Review by adil192 — DISMISSED** [review]

**Pull request merged**